.circleci/config.yml

```diff
@@ -283,7 +284,7 @@ jobs:
             - *ios_gems_cache_key
 
       # Set Ruby Version for chruby
-      - run: echo "ruby-2.4" > .ruby-version
+      - run: echo "ruby-2.7" > .ruby-version
```

Comment:
We could use [pipeline parameters](https://circleci.com/docs/2.0/pipeline-variables/#pipeline-parameters-in-configuration) to define repeated values like `ruby-2.7` and ensure consistency.